docs/reference/contrib/modeladmin/primer.rst

```diff
@@ -35,7 +35,7 @@ while `fields`, `fieldsets`, `exclude` and other attributes you may be used
 to using to configure Django's add/edit views, simply aren't supported by
 Wagtail's version.
 
-'Page type' models need to be treated differently to other models
+'Page type' models need to be treated differently from other models
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Thanks @DanAtShenTech! Please can you add two more characters to the underline here? (Sphinx is very picky about underlines matching the text length...)